### PR TITLE
Ria 3306 hearing notification

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3306

### Change description ###
Changes to send hearing notification to HO

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/HomeOfficeCaseStatus.java [hunk 1]
```diff
@@ -11,13 +11,37 @@
...
+    private String displayAppellantDetailsTitle;
+    private String displayApplicationDetailsTitle;
 
 
     public HomeOfficeCaseStatus(Person person, ApplicationStatus applicationStatus) {
         this.person = person;
         this.applicationStatus = applicationStatus;
     }
 
+    public HomeOfficeCaseStatus(Person person,
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/HomeOfficeCaseStatus.java:23]: Constructor has 10 parameters, which is greater than 7 authorized.

**Review by Lee365 — CHANGES_REQUESTED** [review]
83.5% Coverage to improve

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Comment by jkumar-moj** [comment]
re-opening the PR as it was closed automatically by stale bot

**Pull request merged**